www/includes/easyparliament/templates/html/header.php

```diff
@@ -185,6 +175,17 @@
         </div>
     </nav>
 
+    <?php if (isset($random_banner) && $random_banner ) { ?>
+      <div class="banner">
+          <div class="full-page__row">
+              <div class="banner__content">
+                  <?= $random_banner->content ?>
+                  <a class="button button--small content__button <?= $random_banner->button_class ?>" href="<?= $random_banner->button_link ?>"><?= $random_banner->button_text ?></a>
```

Comment:
It would be better if this would work even if the JSON lacked button_link/text/class and only had content, by testing to see if it had button info.

Reply:
Yep, made this conditional on button_link.